### DISCUSSION TITLE
[Bug]: Unable to match routing to admin (bundle)

### DISCUSSION THREAD

**Discussion by leitge** [pimcore/pimcore#15603] [General]
### Pimcore version

11.0.2

### Steps to reproduce

With a fresh installation of Pimcore 11.0.2, the "new" routing/static routing of the admin prefixes is not recognised. 

The UI throws the following error accordingly:
No route found for "GET http://ec2-18-192-8-101.eu-central-1.compute.amazonaws.com/admin"

First Root Cause Analysis: 
Other routes (pimcore_admin_page_display_preview_image) in "admin-ui-classic-bundle" routing.yaml are recognised. The Symphony "Route Groups and Prefixes" nomenclature also seems to fit. 

Info:
Installation did not throw any errors.
There are no old/other installations on the server. 

Server remains open/accessible so you can have a look at the Symfony logs.

**Comment by leitge** [comment]
<img width="985" alt="Bildschirm­foto 2023-07-03 um 21 30 18" src="https://github.com/pimcore/pimcore/assets/98694299/ba95efd1-7cf1-402c-a19d-df3fe76244de">

**Comment by leitge** [comment]
Update: 
No changed Situation in **11.0.3** - problem remains

Actual URL:
http://ec2-54-93-94-170.eu-central-1.compute.amazonaws.com/admin

**Comment by kingjia90** [comment]
Can you please share the output of the command `bin/console pimcore:bundle:list` ?

AFAIU Code-wise there are no problems, must be some misconfiguration.
Please check if there's any peculiar difference between your local instance with the following files
https://github.com/pimcore/demo/blob/11.0/config/bundles.php
https://github.com/pimcore/demo/blob/11.0/config/routes.yaml
https://github.com/pimcore/demo/blob/11.0/composer.json

**Comment by leitge** [comment]
bundles.php and routes.yaml are different to repo. We changed to default content from GitHub. But the error remain.

We also start and open again the instance, so you can also have a view to the symfony logs:
http://ec2-3-76-46-226.eu-central-1.compute.amazonaws.com/admin

attached the bundles.php, routes.yaml and composer.json which was created after installation. 
![image](https://github.com/pimcore/pimcore/assets/98694299/e9f5a279-5cf3-4ff9-8022-1dfc44306258)


[Pimcore11.zip](https://github.com/pimcore/pimcore/files/12145252/Pimcore11.zip)

**Comment by kingjia90** [comment]
Ahh, looks fine... then what about removing `composer.lock` and try `compose upgrade`? Can't see much from the log in that instance 🤔

**Comment by leitge** [comment]
composer upgrade -> done // no changes

**Comment by cipribucur** [comment]
Hi @leitge can you please provide details about overall infrastructure? 

what server you use ? 
what are the logs from the webserver (nginx, apache, haproxy) say ?
what server configuration you have ?
do you have anything else installed on the system ?
any special configuration worth mentioning ?

Thanks,
Cipri.

**Comment by leitge** [comment]
Hi @cipribucur sure. Let me summarize everything I know about the server. Feel free to ask if you need more specific info.

1. AWS t3a.small: 2vCPUs, 2.0 GiB Memory [Situation did not changed when using a more performant server]

2. OS-Release:
NAME="Amazon Linux"
VERSION="2023"
ID="amzn"
ID_LIKE="fedora"
VERSION_ID="2023"
PLATFORM_ID="platform:al2023"
PRETTY_NAME="Amazon Linux 2023"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2023"
HOME_URL="https://aws.amazon.com/linux/"
BUG_REPORT_URL="https://github.com/amazonlinux/amazon-linux-2023"
SUPPORT_END="2028-03-01"

3. Webserver: NGINX (logs attached)
[nginx_error.log](https://github.com/pimcore/pimcore/files/12145700/nginx_error.log)

4. Nothing else is installed and running on the system

5. No special configuration. New Instance clean instance was used.

**Comment by leitge** [comment]
6. php-v
PHP 8.2.7 (cli) (built: Jul  3 2023 08:45:25) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.7, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.7, Copyright (c), by Zend Technologies

**Comment by leitge** [comment]
7. Pimcore System Req. Check:

![image](https://github.com/pimcore/pimcore/assets/98694299/faa1eef0-46f5-4077-9597-55cc64159837)

**Comment by leitge** [comment]
Last but not least:

8. Database: MySQL 8.0.28 as separate AWS RDS DB Instance

**Comment by fashxp** [comment]
Seems either your container was not build properly or you have some file system cache in place that was not invalidated.
For sure you called already `bin/console cache:clear` multiple times and verified that the container was written properly - also for the environment (dev, prod) you are using? 

It doesn't seem to be a general problem though. So I'm converting it to a discussion.

**Reply by leitge** [reply]
Hi @fashxp,
thanks for you support. 

- cleared and invalidated all filesystem cache 
- run again ` bin/console cache:clear`
- server ist build from scratch. everything seams to run well. a Pimcore 10 installation on a identical server setup runs beautiful.

The problem remains. We do not know adhoc what else we should check. Could it be the php compilation?

br

**Reply by fashxp** [reply]
hmm, just did a clean skeleton installation with our docker containers and it worked like a charm. 

do you file permissions allow accessing following path? 
https://github.com/pimcore/admin-ui-classic-bundle/blob/1.x/config/pimcore/routing.yaml#L2

**Reply by leitge** [reply]
hi @fashxp,

thanks for effort and open minded. 
The routing.yaml is recognized and accessible since also the one fix routine in the yaml is recognize well
<img width="974" alt="image" src="https://github.com/pimcore/pimcore/assets/98694299/e1263615-150c-4645-8824-a9e2be578d73">

The server has also been freshly built several times. The problem has remained. 

br

**Comment by fashxp** [comment]
I mean if `../../src/Controller/Admin/` is also accessible.

**Comment by leitge** [comment]
is accessible and all files available.
It's really a little frustrating. 😕

<img width="858" alt="image" src="https://github.com/pimcore/pimcore/assets/98694299/e2fd9273-4711-4b32-83da-820c8520cbe6">
